examples/cronserver.js

```diff
@@ -0,0 +1,9 @@
+require('distraught').cronServer({
+  crons: [{
+    name: 'Console.log `tick` every 5 seconds',
+    cronTime: '*/5 * * * * *,
```

Comment:
This might be missing the closing quote on the cronTime line.

Reply:
Thanks!